wavefront_sdk/common/proxy_connection_handler.py

```diff
@@ -18,7 +18,7 @@ class ProxyConnectionHandler(connection_handler.ConnectionHandler):
     """
 
```

Comment:
Add `# pylint: disable=too-many-arguments` here.
And add `# pylint: disable=too-many-instance-attributes` before the class, same as: https://github.com/wavefrontHQ/wavefront-sdk-python/blob/007d31405975f8a7668121373961a6261e4e7779/wavefront_sdk/proxy.py#L20